### PR TITLE
completed delete upcoming polls FE,Integration

### PR DESCRIPTION
**Pull request by maniknr-sureify**
completed delete upcoming poll FE and FE/BE integration and made some minor changes in other files

image

image

### CODE REVIEW

**Review by rameshguptha-sureify — APPROVED** [review]

File: src/components/admin/AllUsers.jsx [hunk 1]
```diff
@@ -301,17 +301,23 @@ const AllUsers = () => {
...
+                                no
                             </Button>
                         </Grid>
                         <Grid item>
                             <Button
-                                color="error"
+                                color="success"
                                 variant="contained"
                                 onClick={() => submitUserDelete()}
+                                size="small"
```
Comment (harshavardhan-sureify) [inline src/components/admin/AllUsers.jsx:318]: i think for model cancel and delete would be better

File: src/components/admin/DialogComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            const config = {
+                headers: {
+                    Authorization: `Bearer ${token}`,
+                },
+            };
+            const response = await axios.delete(DELETE_POLL_URL + id, config);
+            if (response.status === 200) {
+                handleToast("Poll deleted successfully", "success");
+                setTimeout(() => {
+                    navigate("activepolls");
```
Comment (harshavardhan-sureify) [inline src/components/admin/DialogComponent.jsx:44]: y navigate
just close the dialog

Comment (harshavardhan-sureify) [reply src/components/admin/DialogComponent.jsx:44]: use refetch instead of navigate

File: src/components/user/PollCard.jsx [hunk 3]
```diff
@@ -52,7 +73,39 @@ export const Poll = ({ activeFlag, poll }) => {
...
+                        sx={{
+                            display: "flex",
+                            justifyContent: "space-around",
+                            width: "5rem",
+                        }}
+                    >
+                        {pollStartDate > currDate && (
+                            <>
+                                <Tooltip title="delete">
+                                    <IconButton onClick={handleClickOpen}>
```
Comment (harshavardhan-sureify) [inline src/components/user/PollCard.jsx:86]: change title to Delete

Comment (maniknr-sureify) [reply src/components/user/PollCard.jsx:86]: done

File: src/components/admin/AllUsers.jsx (hunk 1 above)
Comment (maniknr-sureify) [reply src/components/admin/AllUsers.jsx:318]: resolved

File: src/components/admin/DialogComponent.jsx (hunk 2 above)
Comment (maniknr-sureify) [reply src/components/admin/DialogComponent.jsx:44]: done

**Review by harshavardhan-sureify — APPROVED** [review]

File: src/components/user/EmptyDataContainer.jsx [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
+                flexDirection: "column",
+                width: "100%",
+                height: "60vh",   
+                alignItems: "center",
+                justifyContent: "center",
+                gap: 2,
+            }}
+        >
+            <img
+                src="/assets/notfound.jpeg"
```
Comment (abhishekdundigalla-sureify) [inline src/components/user/EmptyDataContainer.jsx:17]: do not use image path directly, impost and use it.

File: src/components/user/EmptyDataContainer.jsx [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
+                alt="404"
```
Comment (abhishekdundigalla-sureify) [inline src/components/user/EmptyDataContainer.jsx:18]: Better to change alt name, check once

File: src/components/user/ViewSinglePoll.jsx [hunk 6]
```diff
@@ -21,18 +21,34 @@ import { auth } from "../features/User.reducer";
...
     const dispatch = useDispatch();
-    const poll = location.state.pollProps;
-    const isActivePoll = location.state.pollStatus;
-    const options = JSON.parse(poll.options).options;
+    const [poll, setPoll] = useState(null);
+    const [isActivePoll, setIsActivePoll] = useState(false);
+    const [options, setOptions] = useState(null);
+    const [loading, setLoading] = useState(true);
+    let prevSelectedValue;
+    if (options != null) {
```
Comment (abhishekdundigalla-sureify) [inline src/components/user/ViewSinglePoll.jsx:36]: use !==

File: src/components/user/EmptyDataContainer.jsx (hunk 4 above)
Comment (maniknr-sureify) [reply src/components/user/EmptyDataContainer.jsx:17]: done

File: src/components/admin/PollAnswers.jsx [hunk 7]
```diff
@@ -43,7 +56,7 @@ export const PollAnswers = ({ pollId }) => {
         return <LoadingComponent />;
     }
     return (
-        <>
+        <Box sx={{ width: "30rem" }}>
             {rows == null ? (
```
[the remark this answers is not in the document]
Comment (maniknr-sureify) [reply src/components/admin/PollAnswers.jsx:60]: done

File: src/components/user/ViewSinglePoll.jsx (hunk 6 above)
Comment (maniknr-sureify) [reply src/components/user/ViewSinglePoll.jsx:36]: done

File: src/components/user/EmptyDataContainer.jsx (hunk 5 above)
Comment (maniknr-sureify) [reply src/components/user/EmptyDataContainer.jsx:18]: done

**Review by abhishekdundigalla-sureify — APPROVED** [review]

**Pull request merged**